docs/providers/aws/cli-reference/deploy-function.md

```diff
@@ -23,7 +23,12 @@ serverless deploy function -f functionName
 **Note:** This command **now** deploys both function configuration and code by
 default. Just as before, this puts your function in an inconsistent state that
 is out of sync with your CloudFormation stack. Use this for faster development
-cycles and not production deployments
+cycles and not production deployments. Some parts of serverless configuration
```

Comment:
Let's maybe rewrite it to something more like:

During `deploy function`, only changes that can be applied via `updateFunctionCode` and `updateFunctionConfiguration` direct `aws-sdk` calls will take effect. It is important to remember that configuration that is related to separate CloudFormation resources will not be updated. The function that is triggered through an alias will be updated, but the alias itself won't. One instance of that might be when using the `provisionedConcurrency`.